docs/parca-agent-labelling.md

```diff
@@ -0,0 +1,69 @@
+# Labelling
+
+## Metadata labels
+
+The agent does its best to enrich profiles with labels coming from various system facilities relative to the originating process.
+
+### Profiler
+
+* `__name__`: The name of the profiler (e.g. `parca_agent_cpu`).
+* `pid`: The PID of the process.
+
+### Service Discovery
+
+#### Kubernetes
```

Comment:
I think Kubernetes labels should be prefixed with `kubernetes_`, this would make the agent look a little less Kubernetes centric

Reply:
Yes!